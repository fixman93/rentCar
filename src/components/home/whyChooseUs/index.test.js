import React from 'react'
import Adapter from 'enzyme-adapter-react-16'
import { shallow, configure } from 'enzyme'

import { WhyChooseUs } from './index'

configure({ adapter: new Adapter() });

describe('WhyChooseUs', () => {
  const whyChooseUs = shallow(<WhyChooseUs />)
  it('render WhyChooseUs', () => {
    expect(whyChooseUs).toMatchSnapshot()
  })


})