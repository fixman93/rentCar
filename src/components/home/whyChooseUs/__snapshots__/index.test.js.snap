// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WhyChooseUs render WhyChooseUs 1`] = `ShallowWrapper {}`;
